### PR TITLE
Legion Hover Constructor added

### PR DESCRIPTION
**Pull request by EnderRobo**
### Work done
Created the Legion Hovercraft Constructor model, script and unitpic.

Screenshot 2024-12-01 135508

### CODE REVIEW

File: objects3d/Units/legch_clean.cob
Comment (robertthepie) [inline objects3d/Units/legch_clean.cob:1]: Wrong Folder

File: objects3d/Units/legch_clean.bos
Comment (robertthepie) [inline objects3d/Units/legch_clean.bos:1]: Wrong Folder

**Review by robertthepie — CHANGES_REQUESTED** [review]
Can't Load anims as they are in the wrong folders

File: objects3d/Units/legch_clean.cob
Comment (EnderRobo) [reply objects3d/Units/legch_clean.cob:1]: Fixed.

File: objects3d/Units/legch_clean.bos
Comment (EnderRobo) [reply objects3d/Units/legch_clean.bos:1]: Fixed.

**Review by robertthepie — APPROVED** [review]

**Comment by drivver44** [comment]
can i be added to the PR reviewers?

**Comment by ZephyrSkies7** [comment]
@EnderRobo does this still need a unitpic?

**Pull request merged**